src/CONST.js

```diff
@@ -77,7 +77,6 @@ const CONST = {
             TAX_ID_NUMBER: 9,
             SSN: 4,
             ZIP_CODE: 5,
-            PHONE_NUMBER: 15,
```

Comment:
I don't understand why this was moved out of `BANK_ACCOUNT.MAX_LENGTH`. Would you explain that for me?

Reply:
I thought phone number being a generic field we might use it in more than one modules, so I've moved it to root object.

Reply:
Ok, I think that's reasonable 👍